integration_tests/suite/test_wsgi_patch.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2014-2024 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+
+import requests
+from hamcrest import assert_that, contains_string, equal_to
+from wazo_test_helpers.asset_launching_test_case import AssetLaunchingTestCase
+
+ASSET_ROOT = os.path.join(os.path.dirname(__file__), '..', 'assets')
+
+
+class _BaseTest(AssetLaunchingTestCase):
+    assets_root = ASSET_ROOT
+    service = 'wsgi-patch'
+    asset = 'wsgi-patch'
+
+    def test_wsgi_patch(self) -> None:
+        status = self.service_status('wsgi-patch')
+        network_name = f'{self.service}_{self.asset}_default'
+        server_ip = status['NetworkSettings']['Networks'][network_name]['IPAddress']
+        url = f'http://{server_ip}:8080/resource'
+
+        for i in range(150):
```

Comment:
if the goal is to flood the api with requests until it breaks, just use a "while True"

Reply:
If anything goes wrong, I don't want the test to run indefinitely :sweat_smile:

Reply:
yeah, you're right, I guess that's fine. the test will fail if it fails to crash the server after 150 requests, which is reasonable.